README.md

```diff
@@ -1 +1,263 @@
-# builder
+[![Travis Status][trav_img]][trav_site]
+
+Builder
+=======
+
+Builder is a task runner.
+
+Builder is an enhancement to `npm run TASK`.
+
+Builder is a meta-tool for all your common build, quality, and test tasks.
+
+## Overview
+
+At a high level `builder` is a tool for consuming `package.json` `scripts`
+commands, providing sensible / flexible defaults, and support various scenarios
+("archetypes") for your common use cases across multiple projects.
+
+Builder is not opinionated, although archetypes _are_ and typically dictate
+file structure, standard configurations, and dev workflows. Builder supports
+this in an agnostic way, providing essentially the following:
+
+* `NODE_PATH`, `PATH` enhancements to run, build, import from archetypes so
+  dependencies and configurations don't have to be installed directly in a
+  root project.
+* A task runner capable of single tasks (`run`) or multiple concurrent tasks
+  (`concurrent`).
+* An intelligent merging of `package.json` `scripts` tasks.
+
+... and that's about it!
+
+### Usage
+
+To start using builder, install and save `builder` and any archetypes you
+intend to use. For example, here we install `builder` and the
+[builder-react-component][] archetype:
+
+```sh
+$ npm install --save builder builder-react-component
+```
+
+and then edit `.builderrc` like:
```

Comment:
Feature request: generate a `.builderrc` if `builder` and an archetype are installed with `builder init`.